### PR TITLE
chore: cleanup projet

### PR DESCRIPTION
**Pull request by carlh98**
GitHub Issue: #

## Proposed Changes

chore: cleanup projet

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: src/Persistence.Abstractions/Persistence.Abstractions.csproj [hunk 1]
```diff
@@ -1,18 +1,33 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
 	<PropertyGroup>
 		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
```
Comment (jeanplevesque) [inline src/Persistence.Abstractions/Persistence.Abstractions.csproj]: Do we really need that?

File: src/Persistence.Uno.WinUI/Persistence.Uno.WinUI.csproj [hunk 2]
```diff
@@ -1,43 +1,60 @@
...
+		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.3" />
+		<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22000.24" />
+		<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22000.24" />
+	</ItemGroup>
+
+	<PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+		<DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
+	</PropertyGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)'!='net6.0-windows10.0.18362'">
```
Comment (jeanplevesque) [inline src/Persistence.Uno.WinUI/Persistence.Uno.WinUI.csproj]: There are no xaml files in this project, no?

File: src/Persistence.Abstractions/Persistence.Abstractions.csproj (hunk 1 above)
Comment (jeanplevesque) [reply src/Persistence.Abstractions/Persistence.Abstractions.csproj]: Or should that only be for multi-targeted assemblies?

File: src/Persistence.Uno/Persistence.Uno.csproj [hunk 3]
```diff
@@ -1,17 +1,26 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
   <PropertyGroup>
-	<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
-	<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;net461</TargetFrameworks>
-	<TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;monoandroid10.0;monoandroid11.0;uap10.0.19041;net461</TargetFrameworks>
+    <!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
+    <TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;net461</TargetFrameworks>
+    <TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;monoandroid10.0;monoandroid11.0;uap10.0.19041;net461</TargetFrameworks>
```
Comment (vincentcastagna) [inline src/Persistence.Uno/Persistence.Uno.csproj:5]: no `monoandroid12 `?

Comment (carlh98) [reply src/Persistence.Uno/Persistence.Uno.csproj:5]: out of scope for this PR

File: src/Persistence.Uno.WinUI/Persistence.Uno.WinUI.csproj (hunk 2 above)
Comment (carlh98) [reply src/Persistence.Uno.WinUI/Persistence.Uno.WinUI.csproj]: That's correct

File: src/Persistence.Abstractions/Persistence.Abstractions.csproj [hunk 4]
```diff
@@ -9,10 +8,24 @@
 		<PackageId>Nventive.Persistence.Abstractions</PackageId>
 		<Description>Nventive.Persistence.Abstractions</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>persistence;mvvm;uwp;ios;android</PackageTags>
```
Comment (jeanplevesque) [inline src/Persistence.Abstractions/Persistence.Abstractions.csproj]: not sure mvvm is relevant here

File: src/Persistence.Abstractions/Persistence.Abstractions.csproj [hunk 5]
```diff
@@ -9,10 +8,24 @@
 		<PackageId>Nventive.Persistence.Abstractions</PackageId>
 		<Description>Nventive.Persistence.Abstractions</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>persistence;uwp;ios;android</PackageTags>
```
Comment (GabrielChouinardLetourneau) [inline src/Persistence.Abstractions/Persistence.Abstractions.csproj]: Abstactions is used by Uno and Uno.WinUI if I'm not mistaken, shouldn't we include WinUI as well here?

File: src/Persistence.Reactive/Persistence.Reactive.csproj [hunk 6]
```diff
@@ -9,6 +8,15 @@
 		<PackageId>Nventive.Persistence.Reactive</PackageId>
 		<Description>Nventive.Persistence.Reactive</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>persistence;reactive;mvvm;uwp;ios;android</PackageTags>
```
Comment (GabrielChouinardLetourneau) [inline src/Persistence.Reactive/Persistence.Reactive.csproj]: idem

**Review by Guidemarcus — APPROVED** [review]

**Pull request merged**